examples/pes/pswater/input.xml

```diff
@@ -18,7 +19,7 @@
   <system>
     <initialize nbeads='1'> 
       <file mode='xyz' units='angstrom'> h2o.xyz </file>
```

Comment:
This should be avoided: the xyz file should contain "position{angstrom}" in the comment line according to the fastxyz fix...